### DISCUSSION TITLE
Is there a max allowed spend on Plausible?

### DISCUSSION THREAD

**Discussion by Pixelshaped** [plausible/analytics#1728] [Anything Else]
I'm wondering what happens with Plausible if there is malicious traffic on a page (that is reloaded ad libitum). Will the spend grow exponentially or is there a way to cap it?

**Comment by metmarkosaric** [comment]
Hi @Pixelshaped! There is no way to cap the traffic but you will never be charged extra for an occasional traffic spike. There are no surprise fees and your card will never be charged unexpectedly. We haven't experienced any situation like the one you describe yet either as we do our best to exclude any non human / bot traffic by default.

Suppose new higher traffic levels become your new standard and your page views exceed your subscription plan for two consecutive months. In that case, we will contact you and ask you to upgrade your subscription plan to accommodate your new traffic levels using our self-serve platform. 

After we contact you, you will have a week to make a decision. You can decide to continue with a higher plan or to cancel your account at that point.

**Reply by Pixelshaped** [reply]
Thanks!

**Reply by metmarkosaric** [reply]
you're welcome!
